wpeverest/everest-forms
# Enhancement - smart tags added

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds more smart tags for default values.

### How to test the changes in this Pull Request:

1. Create Forms 
2. Add default values to fields using smart tags
3. Verify new tags work as expected.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - smart tags added.
